src/Tools/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/Microsoft.CodeAnalysis.Analyzer.Testing.csproj

```diff
@@ -19,5 +19,6 @@
     <PackageReference Include="Microsoft.VisualBasic" Version="10.0.1" />
     <PackageReference Include="Microsoft.VisualStudio.Composition" Version="15.6.36" />
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+    <PackageReference Include="DiffPlex" Version="1.4.2" PrivateAssets="compile" />
```

Comment:
❓ Why the explicit `PrivateAssets` here?

Reply:
Because DiffPlex is an implementation detail of the test framework -- it's not exposed in the public API, nor would I expect by referencing your nuget package that I'd suddenly get a DiffPlex namespace that I can use. You'd want to be able to replace the DiffPlex dependency with another one if you want without it "breaking" your downstream users.

Reply:
Done